addons/cloudnative-pg/enable

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+set -e
+
+KUBECTL="${SNAP}/microk8s-kubectl.wrapper"
+# shellcheck source=/dev/null
+source "${SNAP}"/actions/common/utils.sh
+
+if [[ "$(arch)" == "aarch64" ]]; then
+	ARCH="arm64"
+else
+	ARCH=$(arch)
+fi
+
+cnpg_get_latest_version() {
+	curl -SsL https://api.github.com/repos/cloudnative-pg/cloudnative-pg/releases/latest | jq -r .tag_name | tr -d 'v'
```

Comment:
We do not want to install the latest version, see https://github.com/canonical/microk8s-community-addons/pull/195#issuecomment-1809627286

Reply:
And how is the addon going to update the version ? I'm guessing that discovering the latest version it's an issue, but what about who's going to update the default version? it's a process that will be done like during the microk8s release? or it can be automated with a PR against this repo ?

Reply:
Will pin the latest stable version by the time of the latest commit and let you know to review! =)

Reply:
As the maintainer of this addon you have to update the version of this addon before every MicroK8s release and in light of important bug fixes.

Reply:
Updating the version shipped with a specific MicroK8s track (eg 1.28) is done by placing a PR against the 1.28 branch of the current repository.